app/assets/javascripts/student_profile_v2/chart.js

```diff
@@ -0,0 +1,77 @@
+(function() {
+  window.shared || (window.shared = {});
+  var dom = window.shared.ReactHelpers.dom;
+  var createEl = window.shared.ReactHelpers.createEl;
+  var merge = window.shared.ReactHelpers.merge;
+
+  var ProfileChartSettings = window.ProfileChartSettings;
+  var HighchartsWrapper = window.shared.HighchartsWrapper;
+
+  // Component for all charts in the profile page.
+  window.shared.Chart = React.createClass({
+    displayName: 'Chart',
+
+    propTypes: {
+      series: React.PropTypes.arrayOf( // you can plot multiple series on the same graph
+        React.PropTypes.shape({
+          name: React.PropTypes.string.isRequired, // e.g. 'Scaled score'
+          data: React.PropTypes.array.isRequired // [year, month, date, value] quads
+        })
+      ),
+      title: React.PropTypes.string.isRequired, // e.g. 'MCAS scores, last 4 years'
+      yAxis: React.PropTypes.object.isRequired // options for rendering the y-axis
+    },
+
+    quadsToPairs: function(quads) {
+      return quads.map(function(quad) {
+        var year = quad[0],
+            month = quad[1],
+            day = quad[2],
+            value = quad[3];
+        return [Date.UTC(year, month - 1, day), value]; // one-index --> zero-index on month
+      });
+    },
+
+    baseOptions: function() {
+      return merge(ProfileChartSettings.base_options, {
+        xAxis: merge(ProfileChartSettings.x_axis_datetime, {
+          plotLines: this.x_axis_bands,
+          min: this.props.timestampRange.min,
+          max: this.props.timestampRange.max
+        })
+      });
+    },
+
+    getDefaultProps: function() {
```

Comment:
This is pretty minor, but I'd suggest moving this method up so it's together with `propTypes`.  In general I think it's helpful practice to keep all the React lifecycle methods up front.